### PR TITLE
Update the deadsnakes ppa location

### PR DESCRIPTION
**Pull request by asottile**

### CODE REVIEW

**Comment by asottile** [comment]
I noticed there's an update-alternatives in the suggested documentation for 14.04.  This is _very_ likely to render the system unusable as there are python libraries as part of the operating system.

I'd instead suggest setting up a [virtualenv](https://virtualenv.pypa.io/en/stable/) and working out of that -- might actually be able to simplify your documentation as well with that.

File: docs/source/install-linux.rst [hunk 1]
```diff
@@ -23,7 +23,7 @@ Ubuntu 14.04
 
 1. Type the following into your Terminal window to install Python 3.5::
 
-    sudo add-apt-repository ppa:fkrull/deadsnakes
```
Comment (shawnblakesley) [inline docs/source/install-linux.rst]: Per the info provided by [deadsnakes](https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa), it should be `sudo add-apt-repository ppa:deadsnakes/ppa`

Comment (asottile) [reply docs/source/install-linux.rst]: heh, I'm the maintainer.  `add-apt-repository ppa:deadsnakes` is shorthand for `ppa:deadsnakes/ppa` btw :)

**Comment by shawnblakesley** [comment]
One last thing, we need anyone contributing to first complete the CLAs mentioned in [CONTRIBUTING.md](https://github.com/anki/cozmo-python-sdk/blob/master/CONTRIBUTING.md) before we can merge it in.

I know it's weird for a small doc update like this, but that's the rules

**Comment by asottile** [comment]
sorry, don't really have the interest in signing a legal document for simple documentation updates.

feel free to carry the patch or not.

**Comment by shawnblakesley** [comment]
I understand.
Closing this PR because we can't accept without the CLA.
Thanks.

**Pull request closed without merging**